src/web/app/pages-session/session-result-page/session-result-page.component.html

```diff
@@ -55,6 +63,12 @@ <h1>
           <tm-student-view-responses *ngIf="responsesForOtherRecipient.length" [responses]="responsesForOtherRecipient" [feedbackQuestion]="question.feedbackQuestion" [timezone]="session.timeZone" [statistics]="question.questionStatistics"></tm-student-view-responses>
         </div>
         <div id="given-responses" class="mt-4" *ngIf="question.responsesFromSelf.length">
+          <div class="other-response" *ngIf="!question.feedbackQuestion.showResponsesTo.includes(visibilityRecipient)">
```

Comment:
this is parked in the wrong place - you just need to do an else off this `<div id="other-responses" *ngIf="canStudentSeeResponses(question)">` on line 57. 

You also do not need the `question.feedbackQuestion.showResponsesTo.includes(visibilityRecipient)`, we can use `canStudentSeeResponses`

If not you will have issues like this where I set the responses to be actually anonymously visible to STUDENTS:

<img width="1077" alt="Screenshot 2021-07-25 at 1 10 33 PM" src="https://user-images.githubusercontent.com/10579415/126888486-eff0f769-f1d9-4b7a-97b9-ca1b88d20a2b.png">

Reply:
Ah, didn't know we have this function `canStudentSeeResponses`. Thanks for the feedback, I will update it.